cmake/modules/FindOpenTracing.cmake

```diff
@@ -0,0 +1,10 @@
+find_path(OpenTracing_INCLUDE_DIRS opentracing/tracer.h)
+find_library(OpenTracing_LIBRARIES libopentracing.so)
+
+if(EXISTS ${OpenTracing_INCLUDE_DIRS} AND EXISTS ${OpenTracing_LIBRARIES})
+    message(STATUS "FOUND " ${OpenTracing_INCLUDE_DIRS} " " ${OpenTracing_LIBRARIES})
+    set(OpenTracing_FOUND true)
+    add_library(OpenTracing INTERFACE)
```

Comment:
@suab321321 you might need to specify to add opentracing library and includedirs in this target. Please refer previous version of code

Reply:
@ideepika I m doing this in one place inside `ExternalProjectHelper.cmake` which is called from `IncludeJaeger.cmake`